From 0eb3f167bc65fa9bf30926e0f0b46db670b4d434 Mon Sep 17 00:00:00 2001
From: zachary <zhangzg@marvell.com>
Date: Tue, 20 Dec 2016 21:39:09 +0800
Subject: [PATCH 1197/2241] dts: spi: a3700: use gic/irq macro in spi dt node

- Currently in SPI DT node of Armada3700 dtsi file, interrupt property
  are hard coded number, instead of well-known GIC/IRQ Macro which all
  other DT nodes are using.
- This patch replace hard coded number in SPI DT node with GIC/IRQ marcos,
  to make it more clean and user friendly.

Change-Id: Id8d27627d1c05c1f0c3d17fc602fc641129ab33d
Signed-off-by: zachary <zhangzg@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/34784
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Reviewed-by: Hua Jing <jinghua@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 2bb64e1..9895cab 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -113,7 +113,7 @@
 				clock-frequency = <160000>;
 				max-frequency = <40000>;
 				fifo-mode;
-				interrupts = <0 0 4>;
+				interrupts = <GIC_SPI 0 IRQ_TYPE_LEVEL_HIGH>;
 				status = "disabled";
 			};
 
-- 
2.7.4

